fortuna/src/backend/routes/api/auth.js

```diff
@@ -0,0 +1,86 @@
+const express = require('express');
+const router = express.Router();
+const auth = require('../middleware/auth');
+const jwt = require('jsonwebtoken');
+const { check, validationResult } = require('express-validator');
+const config = require('config');
+const bcrypt = require('bcrypt');
+
+
+const User = require('../../../models/userModel');
+
+
+router.get('/', auth, async (req, res) => {
+    try{
+        const user = await User.findById(req.user.id).select('-password');
+        res.json(user);
+    } catch(err) {
+        console.error(err.message);
+        res.status(500).send('Server Error');
+    }
+});
+
+
+// Authenticate user & get token
+// Public
+router.post('/', [
+    check('userName', 'Please enter a username with 3 or more characters')
+        .isLength(3),
+    check('password', 'Password is required')
+        .exists()
+    ], 
+    async (req, res) => {
```

Comment:
Ditto: can we get types on req and res so stupid people like me know what's going on?